packages/fast-usdc/src/exos/transaction-feed.js

```diff
@@ -118,10 +118,12 @@ export const prepareTransactionFeedKit = (zone, zcf) => {
         /**
          * Add evidence from an operator.
          *
+         * NB: the operatorKit is responsible for
```

Comment:
for... what?

Reply:
I must have gotten a Slack notification!

> The operatorKit is responsible for preventing a disabled kit from attesting.

I'll get that in before merge. I'll probably wait for another reviewer's comments before pushing again.